Leetcode/mereTwoLiat/mergeTwoLists.cpp

```diff
@@ -0,0 +1,23 @@
+//
+//  mergeTwoLists.cpp
+//  Leetcode
+//
+//  Created by dsh on 2021/7/11.
+//
+
+#include "mergeTwoLists.hpp"
+
+ListNode* Solution::mergeTwoList(ListNode* l1,ListNode* l2) {
+    if (l1 == nullptr)
```

Comment:
指针可以像 bool 那样直接判定是否为空
```suggestion
    if (!l1)
```